Packs/Tenable_io/Integrations/TenableioEventCollector/TenableioEventCollector.py

```diff
@@ -201,6 +200,13 @@ def insert_type_to_logs(audit_logs: list, vulnerabilities: list):
         log.update({'xsiam_type': 'vulnerability'})
 
 
+def call_send_events_to_xsiam(events, vulnerabilities):
+    """Enhanced and sends events and vulnerabilities to XSIAM"""
+    insert_type_to_logs(audit_logs=events, vulnerabilities=vulnerabilities)
```

Comment:
I see this one was called regardless of the `should_push_events` value, can you double check?

Reply:
`call_send_events_to_xsiam` is called in `main` only when `should_push_events` is True

Reply:
I know, but `insert_type_to_logs` is called anyway

Reply:
fixed